### ISSUE TITLE
Return Url not updating 

### ISSUE THREAD

**Issue by cm-creativecentral** [flutter-stripe/flutter_stripe#2023]
> https://github.com/flutter-stripe/flutter_stripe/blob/main/example/lib/screens/regional_payment_methods/ideal_screen.dart
> 
> and
> 
> https://docs.flutter.dev/ui/navigation/deep-linking

![image](https://github.com/user-attachments/assets/510875a9-d2a3-40de-a8ca-c2d43c42ce00)
after passing return url still it was taking defalu stripesdk:// url on andriod but on ios it was working correctly.
i have checked deeplinking also working file on andriod but return url is not setting properly

![image](https://github.com/user-attachments/assets/0394f5fd-372c-47cc-9054-82f7eb1c58ea)

_Originally posted by @cm-creativecentral in https://github.com/flutter-stripe/flutter_stripe/discussions/1999#discussioncomment-11684131_

**Comment by cm-creativecentral** [comment]
@remonh87 can you help me with this ?

**Comment by remonh87** [comment]
are you using android? because on android you need to supply the url  as part of the init settings in stripe.  

https://github.com/flutter-stripe/flutter_stripe/blob/b76793a593ae271dca082b10d6857a3a34f679fb/example/lib/main.dart#L13

**Comment by cm-creativecentral** [comment]
Yes i have added that still it takeing 'flutterstripe....' as an return url and it was not redirect to that retun url
![image](https://github.com/user-attachments/assets/816fc1ef-39bb-419c-aed6-ac4430855f7d)

**Comment by cm-creativecentral** [comment]
> are you using android? because on android you need to supply the url as part of the init settings in stripe.
> 
> https://github.com/flutter-stripe/flutter_stripe/blob/b76793a593ae271dca082b10d6857a3a34f679fb/example/lib/main.dart#L13


Yes i have added that still it takeing 'flutterstripe....' as an return url and it was not redirect to that retun url

**Comment by cm-creativecentral** [comment]
@remonh87  can you help me with this ?

**Comment by remonh87** [comment]
Also do not forget to call apply settings

**Comment by cm-creativecentral** [comment]
> Also do not forget to call apply settings
yes i called apply settings also please check above screenshot

https://github.com/flutter-stripe/flutter_stripe/issues/2023#issuecomment-2568705673

**Comment by cm-creativecentral** [comment]
@remonh87 can you help me with this ?

**Comment by cm-creativecentral** [comment]
@remonh87 any update on this issue ?

**Comment by cm-creativecentral** [comment]
@remonh87  any update on this issue. how we can fix that ?

**Comment by shebnik** [comment]
I encountered probably the same issue on Android where the app wasn't returning after Stripe Link/CashApp payments. The solution was to modify the AndroidManifest.xml file. Specifically, removing `android:taskAffinity=""` from the activity tag resolved the problem for me.

**Comment by cm-creativecentral** [comment]
> I encountered probably the same issue on Android where the app wasn't returning after Stripe Link/CashApp payments. The solution was to modify the AndroidManifest.xml file. Specifically, removing `android:taskAffinity=""` from the activity tag resolved the problem for me.

It worked, but the return URL is still the SDK's default. However, everything is functioning correctly now.

**Comment by cm-creativecentral** [comment]
@shebnik  thanks for help

**Comment by alvar-dev** [comment]
> I encountered probably the same issue on Android where the app wasn't returning after Stripe Link/CashApp payments. The solution was to modify the AndroidManifest.xml file. Specifically, removing `android:taskAffinity=""` from the activity tag resolved the problem for me.

That worked. Thank you!
